### DISCUSSION TITLE
NX mitosis plugin - Be able to generate consumable build outputs from the libs folder across multiple apps

### DISCUSSION THREAD

**Discussion by algoflows** [BuilderIO/jsx-lite#1053] [Ideas]
NX mitosis plugin - Be able to generate consumable build outputs from the libs folder across multiple apps?!

Would love to see an official NX plugin for Mitosis as NX is becoming the defacto way to build out multiple apps and libs in a monorepo... I am increasingly looking for NX plugin support before using and choosing to work with a framework or libs.

**Comment by tomByrer** [comment]
TBH I think I'd prefer anything over Yarn workspaces....
